models/core/liquidity_pools.sql

```diff
@@ -15,6 +15,39 @@ dfk_lp as (
     from {{ ref('dfk_dex_lp_labels') }}
 ),
 
+src_logs_lp as (
+select 
+    event_inputs:pair::string as pool_address,
+    '' as pool_name,
+    event_inputs:token0::string as token0,
+    event_inputs:token1::string as token1
+from {{ ref('logs') }} 
+Where event_name = 'PairCreated'
+  ),
+  
+  logs_lp as (
+  select 
+    pool_address,
+    concat(' ', concat_ws('-',t0.token_symbol, t1.token_symbol), ' LP') as pool_name,
```

Comment:
any reason for the extra concat white space in the front?

Reply:
Without the space in front of the 'LP', there was no space between token1 and "LP".  Because multiple concats, I changed the line to the below to make it more readable:

t0.token_symbol || '-' || t1.token_symbol || ' LP' as pool_name, -- "TOKEN0-TOKEN1 LP"